src/components/select/index.tsx

```diff
@@ -1,27 +1,20 @@
 import * as React from 'react';
-import { Props as SelectProps } from 'react-select';
 import { useTheme, useMode } from 'utils/hooks';
 import { Row, Col } from 'components';
-import { InputComponent } from '../../utils/types';
 import { StyledSelectWrapper } from './style';
 import ChipsComponent from './chips-component';
-import SelectComponent from './select-component';
+import SelectComponent, { ComponentProps } from './select-component';
 
-export interface SelectInputProps extends SelectProps, InputComponent {
-  value?: any;
-  selectWrapperProps?: React.HTMLAttributes<HTMLDivElement>;
+export interface SelectInputProps extends ComponentProps {
   clearChips?: boolean;
-  wrapperSelectSizes?: {
-    lg?: number | string;
-    md?: number | string;
-    sm?: number | string;
-  };
   hideChips?: boolean;
 }
 
 export const SelectInput = React.forwardRef<HTMLDivElement, SelectInputProps>(
   (
     {
+      async,
+      creatable,
```

Comment:
@yurii-korolchuk What's the purpose of taking out async and creatable from `...props`?  `async` and `creatable` are not used here. You pass `...props` in `SelectComponent` already.

Reply:
@vlacher12 removed